### DISCUSSION TITLE
番剧播放器core.js启用字幕设置

### DISCUSSION THREAD

**Discussion by Howard20181** [the1812/Bilibili-Evolved#3653] [功能建议 / Ideas]
发现现在投稿视频和番剧都是用https://s1.hdslb.com/bfs/static/player/main/core.js 播放
在普通投稿的情况下如果有CC字幕可以设置字幕的样式
![image](https://user-images.githubusercontent.com/40033067/188262397-58488808-6cba-417c-862b-59988b16782c.png)
如https://www.bilibili.com/video/BV1Gf4y1y7wc

但是番剧就直接砍掉这个功能了
![image](https://user-images.githubusercontent.com/40033067/188262417-648a77b5-2c87-4fc6-8d4a-90e94f295726.png)
如https://www.bilibili.com/bangumi/play/ss39044/
调用的是同一个播放器，应该有办法在播放番剧时打开字幕设置

**Comment by the1812** [comment]
有办法也是 b 站有办法, 脚本不能修改这些资源文件的内容
